### PR TITLE
Added expressions and tests

### PR DESCRIPTION
**Pull request by SND96**
Added more comparison expressions, arithmetic expressions and logical expressions. Added tests for each of the new expressions.

### CODE REVIEW

File: src/expression/arithmetic_expression.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        children = []
+        if left is not None:
+            children.append(left)
+        if right is not None:
+            children.append(right)
+        super().__init__(exp_type, rtype=ExpressionReturnType.BOOLEAN,
+                         children=children)
+
+    def evaluate(self, *args):
+        vl = self._children[0].evaluate(*args)
```
Comment (gaurav274) [inline src/expression/arithmetic_expression.py]: Avoid using _children. use get_child

File: src/expression/logical_expression.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        if left is not None:
+            children.append(left)
+        if right is not None:
+            children.append(right)
+        super().__init__(exp_type, rtype=ExpressionReturnType.BOOLEAN,
+                         children=children)
+
+    def evaluate(self, *args):
+        # print(*args)
+        if len(self._children) == 2 :
```
Comment (gaurav274) [inline src/expression/logical_expression.py]: Use get child_count. use get_child

File: src/expression/logical_expression.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
+from .abstract_expression import AbstractExpression, ExpressionType, \
```
Comment (gaurav274) [inline src/expression/logical_expression.py]: rewrite this import. use src/....

**Comment by SND96** [comment]
@gaurav274 Made the changes to _children and replaced with get_child functions. Also modified the import statements to be global.

**Pull request merged**